src/index.js

```diff
@@ -286,18 +286,14 @@ class Radar {
       .catch(handleError(callback));
   }
 
-  static ipGeocode(arg0, arg1=defaultCallback) {
```

Comment:
I think this is technically a breaking change. 

It's probably best to leave the initial arg in place, but do a `console.warn` in the case where `arg0` as an object is provided.
```js
console.warn('Radar SDK: ipGeocode parameters have been deprecated.');
```

Reply:
updated, can you take a look to see if it's what you had in mind?

Reply:
Yep, looks good.

Reply:
Cool. Good to merge?